test/test_main.py

```diff
@@ -206,59 +208,71 @@ def test_multiple_events_handled(self):
             'USE_ACTIONS': True,
         }))
 
-        fake_script(self, 'action-get', "echo '{}'")
-
         # Sample events with a different amount of dashes used
         # and with endpoints from different sections of metadata.yaml
         events_under_test = [(
+            [],
             EventSpec(InstallEvent, 'install', charm_config=charm_config),
             {},
         ), (
+            [],
             EventSpec(StartEvent, 'start', charm_config=charm_config),
             {},
         ), (
+            [],
```

Comment:
all of these [] very much point me to say:
lets not do generic argument tests, because all of this is just noise. If you want a table test create a small type that lets you describe the things you are passing, and not pass the ones you don't need.
```
events_under_test = [{
  spec: EventSpec...
  }, {
  spec: EventSpec(...)
  expected_events=[...]
  ]
```
etc

I find it really hard to map what is in the spec in a table far away from where that table entry is being evaluated.

Reply:
Fixed in #132 by adding a separate test case and leaving this one unchanged per our discussion. Also merged into this PR.